firmware/rusefi.cpp

```diff
@@ -84,17 +84,28 @@
  * 2.777ms, so we schedule spark firing at '2.777ms after the 690 position sensor event', thus combining trigger events
  * with time-based offset.
  *
+ * @section tunerstudio Getting Data To and From Tunerstudio
+ *
+ * Contains the enum with values to be output to Tunerstudio.
+ * console/binary/tunerstudio_outputs.h
+ * Tunerstudio output channels are handled completely manually, as opposed to settings where offsets are managed automatically by the gen_config scripts.
```

Comment:
this is no longer true, have a look at `output_channels.txt`!

Reply:
And offsets aren't handled manually anymore?

Reply:
make a change in `output_channels.txt`, run `gen_output_channels.sh` and see the magic!